src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -11020,7 +11020,7 @@ private ExpressionSyntax ParseExpressionContinued(ExpressionSyntax unaryOrPrimar
                 // precedence level.  Examples include binary operator, assignment operators, range operators `..`, as
                 // well as `switch` and `with` clauses.
 
-                var (operatorTokenKind, operatorExpressionKind) = getOperatorTokenAndExpressionKind();
+                var (operatorTokenKind, operatorExpressionKind) = GetOperatorTokenAndExpressionKind();
```

Comment:
i would rename this to GetExpressionOperatorTokenAndKind.  When these were local functions, it was very clear what context this was relating to.  Now that they're floating, having 'Expression' be prominent will help clear up where this is intended to be used for me.

Reply:
I think `TokenAndKind` suffix specifically, muddies the waters a bit, because this method doesn't return any token, only SyntaxKinds. I will try and suggest a more clear name to change this to.

Reply:
Renamed.